artifacts/phar/Makefile

```diff
@@ -0,0 +1,20 @@
+# Build phar
+build-phar: test
+	@echo Copying sources
+	@mkdir -p /tmp/phpmetrics-build
+	@cp * -R /tmp/phpmetrics-build
+	@rm -Rf /tmp/phpmetrics-build/vendor /tmp/phpmetrics-build/composer.lock
+
+	@echo Releasing phar
+	@sed -i "s/<VERSION>/`semver tag`/g" /tmp/phpmetrics-build/artifacts/phar/build.php
+
+	@echo Installing dependencies
+	@cd /tmp/phpmetrics-build && composer install --no-dev --optimize-autoloader --prefer-dist
```

Comment:
I believe you can use `composer install --no-dev --optimize-autoloader --prefer-dist -d /tmp/phpmetrics-build`